Tests/src/PublicAPITests/SpaceSystemTests.cpp

```diff
@@ -279,6 +279,24 @@ void GetSpacesMetadata(::SpaceSystem* SpaceSystem, const Array<String>& SpaceIds
 	OutMetadata = Result.GetMetadata();
 }
 
+void GetSpaceTags(::SpaceSystem* SpaceSystem, const String& SpaceId, Array<String>& OutTags)
+{
+	auto [Result] = AWAIT_PRE(SpaceSystem, GetSpaceMetadata, RequestPredicate, SpaceId);
+
+	EXPECT_EQ(Result.GetResultCode(), csp::systems::EResultCode::Success);
+
+	OutTags = Result.GetTags();
```

Comment:
[fix] Avoid out params unless necessary, prefer return.

Reply:
This is essentially a copy of GetSpace[s]Metadata. would you like me to change both?

Reply:
Ah I see, and I see there's an argument (a weak one imo in test code) to do it this way, to avoid a double construct throughout the test.

I'll drop this to a [Consider], although if it were me I would change both to be return base. The less mutation the better, I want as few possible reasons for the tests to go wrong as possible.

Reply:
done in ac6f3d3006b386a3f496bff7fee3023e0ead9334